test/unit/core/dom/layout/test-viewport-observer.js

```diff
@@ -82,21 +79,28 @@ describes.sandboxed('DOM - layout - Viewport Observer', {}, (env) => {
 
     it('observed element should have its callback fired each time it enters/exist the viewport.', () => {
       const viewportEvents = [];
-      observeWithSharedInOb(el1, (inViewport) =>
+      observeIntersections(el1, (inViewport) =>
         viewportEvents.push(inViewport)
       );
```

Comment:
It looks like you're testing properties on the entry, so let's rename accordingly:
```suggestion
      observeIntersections(el1, (entry) =>
        viewportEvents.push(entry)
      );
```

Reply:
Done.